### PR TITLE
added clearer instructions, examples for adding stylesheets

### PR DESCRIPTION
**Pull request by ghost**
Again, inspired by #6, but splitting out the edit, rewording it a bit, and adding more examples.

I know I found this really confusing when I started with hugo/this theme so hopefully this helps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,35 @@ Clone this repository into your `themes` folder of your site. Issue the followin
...
 	static/css/layout.css
+	static/css/colors.css
 
-### Add Color
-Add CSS to color your site into:
+And add it to your config under `.params.css`:
 
-	static/css/color.css
+#### TOML example:
+
```
Comment (simonmika) [inline README.md:24]: It should use a markdown code block: https://help.github.com/articles/creating-and-highlighting-code-blocks/

File: README.md [hunk 2]
```diff
@@ -10,15 +10,35 @@ Clone this repository into your `themes` folder of your site. Issue the followin
...
+	[params]
+		...
+		css = ["layout.css", "colors.css"]
+		...
+
+#### YAML example:
+
```
Comment (simonmika) [inline README.md:33]: It should use a markdown code block: https://help.github.com/articles/creating-and-highlighting-code-blocks/

File: README.md [hunk 3]
```diff
@@ -10,15 +10,35 @@ Clone this repository into your `themes` folder of your site. Issue the followin
...
+	params:
+		...
+		css: ["layout.css", "colors.css"]
+		...
+
+#### JSON example:
+
```
Comment (simonmika) [inline README.md:42]: It should use a markdown code block: https://help.github.com/articles/creating-and-highlighting-code-blocks/

**Review by simonmika — CHANGES_REQUESTED** [review]
Thanks, great PR. Could you just add some code blocks as well?

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:24]: is that better? That's so weird, I was sure indented code blocks are valid Markdown, but those docs make no mention of it!

**Review by simonmika — APPROVED** [review]

**Pull request merged**